File: C/FizzBuzz.c
# include <stdio.h>
int main() {
    int a;
    printf("Enter the number:\n");
    scanf("%d",&a);
    if(a%3==0 && a%5==0)
    printf("FizzBuzz");
    else if (a%3==0)
    printf("Fizz");
    else if (a%5==0)
    printf("Buzz");
    else
    printf("%d",a);
    
    return 0;a
}

File: Python/fizzbuzz.py
# Fizz Buzz Program
# Hacktoberfest 2k19

userInput = int(input('Enter a number: '))

for i in range(1, userInput + 1):
    if i % 3 == 0 and i % 5 == 0:
        print('Fizz Buzz')
    elif i % 3 == 0:
        print('Fizz')
    elif i % 5 == 0:
        print('Buzz')
    else:
        print(i)
